sdcm/cluster.py

```diff
@@ -5093,7 +5094,7 @@ def get_node_ip_list(verification_node):
                 return None
 
         target_node_ip = node.ip_address
-        undecommission_nodes = [n for n in self.nodes if n != node]
+        undecommission_nodes = [n for n in self.nodes if n != node and not n.decommissioning]
```

Comment:
how about using `self.running_nemesis` attr for that - it's quite common pattern